sycl/source/builtins_generator.py

```diff
@@ -641,12 +641,14 @@ class Def(DefCommon):
   def __init__(self, return_type, arg_types, invoke_name=None,
                invoke_prefix="", custom_invoke=None, fast_math_invoke_name=None,
                fast_math_custom_invoke=None, convert_args=[], size_alias=None,
-               marray_use_loop=False, template_scalar_args=False):
+               marray_use_loop=False, template_scalar_args=False,
+               deprecation_message=None):
     super().__init__(return_type, arg_types, invoke_name, invoke_prefix,
                      custom_invoke, size_alias, marray_use_loop,
                      template_scalar_args)
     self.fast_math_invoke_name = fast_math_invoke_name
     self.fast_math_custom_invoke = fast_math_custom_invoke
+    self.deprecation_message = deprecation_message
```

Comment:
Since this isn't passed to the superclass, we should add a note about it in the comment above the `__init__` definition.